test/paper_tooltip_test.dart

```diff
@@ -0,0 +1,206 @@
+@TestOn('browser')
+library polymer_elements.test.paper_tooltip_test;
+
+import 'dart:async';
+import 'dart:html';
+import 'package:polymer_elements/paper_tooltip.dart';
+import 'package:web_components/web_components.dart';
+import 'package:test/test.dart';
+import 'common.dart';
+import 'sinon/sinon.dart' as sinon;
+
+/**
+ * Original tests:
+ * https://github.com/PolymerElements/paper-tooltip/tree/master/test
+ */
+
+main() async {
+  await initWebComponents();
+
+  bool isHidden(element) {
```

Comment:
As long as its local to this file, I don't think its an issue to have it named isHidden/isNotHidden. I don't feel strongly though, if you can come up with a name that makes more sense (I can't think of one off the top of my head).